system-addon/common/Reducers.jsm

```diff
@@ -18,12 +15,6 @@ const INITIAL_STATE = {
   App: {
     // Have we received real data from the app yet?
     initialized: false,
-    // The locale of the browser
-    locale: "",
```

Comment:
This is set in `TelemetryFeed`, we probably want to have that send `Services.locale.getRequestedLocale()` instead

Reply:
Oh doh. I looked at `TelemetryFeed` too quickly. It said `appInfo` and I assumed it was `Services.appinfo`.

Reply:
Switched to `Services.locale.getRequestedLocale()` and updated test.